hrm_oo.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import numpy as np
+from readHR import EcgReader
+from hrmcalcs2oo import hrmcalcs
+from hrmtb import TachyBrady
+
+
+class HrmOutput:
+
+    def __init__(self, file='Test_ECG.csv', start_min=3, end_min=5,
+                 brady_limit=60, tachy_limit=100):
+        self.extract_vals(file, start_min, end_min, brady_limit, tachy_limit)
+        self.file = file
+        self.print_hrmoutput()
+
+    def extract_vals(self, file, start_min, end_min, brady_limit, tachy_limit):
```

Comment:
If those input variables were mapped to object attributes, then they would come into this method for "free" as part of `self`.